dannythedeveloper1/react-challenge-project-sep-trial
# Dk 7 update dependencies

## Changes

1. `package.json` and `package-lock.json` 
2. dependencies updated to latest version and installed

## Testing Steps

1. run `npm outdated` in the terminal to check if its not on latest version

Closes #7
